src/common/TracepointProvider.h

```diff
@@ -56,7 +56,7 @@ class TracepointProvider : public md_config_obs_t, boost::noncopyable {
 
   template <const Traits &traits>
   static void initialize(CephContext *cct) {
-#if WITH_LTTNG
+#ifdef WITH_LTTNG
```

Comment:
shouldn't need to change this header after fixing the `#cmakedefine WITH_LTTNG 1`